styles.css

```diff
@@ -1,3 +1,131 @@
 * {
 	box-sizing: border-box;
+}
+body {
+	background-color: #CCCCFF;
+	background-image: url('img/ship-7679100.jpg');
+	background-size: cover;
+	height: 100vh;
+}
+
+.navTop {
+	display: flex;
+	justify-content: space-around;
+	background-color: #FDBCB4;
+	margin: 0px 0px 20px 0px;
+	align-items: center;
+	position: fixed;
+	overflow: hidden;
+    top: 0;
+    width: 100%;
+	font-family: "Courier New";
```

Comment:
- font-family could possibly go into the body section if all fonts are the same
- should add a secondary font e.g. sans for accessibilty